docs/resources/templates/Design-Document-Template.md

```diff
@@ -0,0 +1,52 @@
+Design Title: 
+Authors: 
+Date: 
+
+* Table of Contents
+{:toc}
+
+## Overview
+This section explains the motivation of the work, providing the status QUO and the problems associated with it. This section also proposes the high level solution to the problem.
+
+##	Goals
```

Comment:
any reason for having so many spaces between `##` and the title of the sections? This applies to all headers. One space should be sufficient?

Reply:
I realized I accidentally used tabs.